src/components/views/elements/ReplyThread.tsx

```diff
@@ -231,37 +238,38 @@ export default class ReplyThread extends React.Component<IProps, IState> {
         return replyMixin;
     }
 
-    public static makeThread(
-        parentEv: MatrixEvent,
-        onHeightChanged: () => void,
-        permalinkCreator: RoomPermalinkCreator,
-        ref: React.RefObject<ReplyThread>,
-        layout: Layout,
-        alwaysShowTimestamps: boolean,
-    ): JSX.Element {
-        if (!ReplyThread.getParentEventId(parentEv)) return null;
-        return <ReplyThread
-            parentEv={parentEv}
-            onHeightChanged={onHeightChanged}
-            ref={ref}
-            permalinkCreator={permalinkCreator}
-            layout={layout}
-            alwaysShowTimestamps={alwaysShowTimestamps}
-        />;
+    public static hasThreadReply(event: MatrixEvent) {
+        return Boolean(ReplyThread.getParentEventId(event));
     }
 
     componentDidMount() {
         this.initialize();
+        this.setQuotedExpandable();
     }
 
     componentDidUpdate() {
         this.props.onHeightChanged();
+        this.setQuotedExpandable();
     }
 
     componentWillUnmount() {
         this.unmounted = true;
     }
 
+    private setQuotedExpandable() {
```

Comment:
There's opportunity for this function to not set the expandable flags - would suggest naming it `trySetQuoteExpandable` or similar. 

Also is it intentional that this is "quoted" but the function it eventually calls is "quote" ?